packages/cli/src/service/job.ts

```diff
@@ -48,7 +48,7 @@ export async function log(id: string, opts: any): Promise<void> {
   const client = initClient(opts.hostIp, opts.port);
   try {
     const log = await client.job.log(id);
-    logger.info(log);
+    logger.info(JSON.stringify(log));
```

Comment:
Why change this?

Reply:
A bug here, the parameter of `logger.info` should be `string`.
Test case is on the way.